<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sports</class>
 <widget class="QDialog" name="Sports">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>1270</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="prevPage">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>550</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Zilla Slab</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPage">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>550</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Zilla Slab</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentPage">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>560</y>
     <width>47</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>544</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rows</string>
   </property>
  </widget>
  <widget class="QComboBox" name="rowsCount">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>69</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>500</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>580</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Count </string>
   </property>
  </widget>
  <widget class="QLabel" name="totalCount">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>585</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>520</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>510</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Search</string>
   </property>
   <widget class="QRadioButton" name="vbet">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VBET</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="pinnacle">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PINNACLE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="megapari">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MEGAPARI</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="marathon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MARATHON</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="all">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>All Show</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount($)</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalAmount">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>515</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>540</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Win      ($)</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalWin">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>545</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="status">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>580</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalLoss">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>575</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>570</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Loss      ($)</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalBenefit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>605</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>600</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Benefit  ($)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
